impexp-client/src/main/java/org/citydb/gui/modules/exporter/preferences/CityGMLExportPreferences.java

```diff
@@ -37,13 +37,18 @@
 	public CityGMLExportPreferences(Config config) {
 		super(new CityGMLExportEntry());
 		
-		root.addChildEntry(new DefaultPreferencesEntry(new VersionPanel(config)));
```

Comment:
Well, this is a good but not so easy question. Let me try to explain my thoughts for the current implementation:

The CityGML version is not only an output format option. In the config, it is rather stored in the query options and thus has an impact on the SQL query that is sent to the database. It makes a difference whether you query the database with CityGML 1.0 or 2.0 because, for instance, bridges and tunnels are excluded when using CityGML 1.0. And this directly affects the output of SQL query operations like `count` or `limit`. 

For example: Assume you have a database with 50 bridges and 50 buildings. If you would query all CityGML 2.0 features from the database but use CityGML 1.0 as format option for the output file, then the exporter would say that it found 100 features matching the query (this is the result of the SQL query) but only 50 are written to the file. So there is a mismatch of numbers. Likewise, if you use a counter filter to only export the first 10 features and assume that the database would only return 10 bridges, then the exporter would say that it found and will export 10 features (based on the SQL query) but 0 features are written to the file. 

Or assume you use the following XML query for an export from the above database:

```xml
<query xmlns="http://www.3dcitydb.org/importer-exporter/config">
  <typeNames>
    <typeName xmlns:core="http://www.opengis.net/citygml/1.0">core:_CityObject</typeName>
  </typeNames>
</query>
```

This will, of course, only export the 50 buildings for a CityGML file. What would you expect for the CityJSON file with the same query? 50 buildings and 50 bridges? But bridges are not part of CityGML 1.0. So, how could a query using `{http://www.opengis.net/citygml/1.0}_CityObject` as feature type filter ever return bidges?

Well, for these reasons, I kept the CityGML version as a general option because this option not only affects the output format but also the data model used for the query and the export. I'm happy for any good proposals how we could resolve this in a different way.

Reply:
Yes, you are right, the key problem ist that CityGML ist not only an encoding, but also a data model. So, the CityGML version should be kept as a general option for CityGML and CityJSON export.

Reply:
Ok, I will add a hint to the CityGML 2.0 version option that it should be used for CityJSON exports to get all feature types.

Reply:
Do you think this hint would be helpful for users? Or should we just explain it in the user manual?

![image](https://user-images.githubusercontent.com/5517439/106428188-da7e0380-6468-11eb-9bc1-5a087dbf3c40.png)

Reply:
The hint looks good. We also have to explain it in the documentation to tell why the option  _Use CityGML 1.0_ is not recommended.

Reply:
added with 896d7337f357aec97e4d1d660e304e9c73b53189